src/mono/sample/wasm/browser-bench/Console/Wasm.Console.Bench.Sample.csproj

```
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <WasmCopyAppZipToHelixTestDir Condition="'$(ArchiveTests)' == 'true'">true</WasmCopyAppZipToHelixTestDir>
-    <WasmMainJSPath>$(MonoProjectRoot)\wasm\test-main.js</WasmMainJSPath>
+    <WasmMainJSPath>$(MonoProjectRoot)\browser\test-main.js</WasmMainJSPath>
```

Can we use `BrowserProjectRoot`?

```suggestion
    <WasmMainJSPath>$(BrowserProjectRoot)\test-main.js</WasmMainJSPath>
```